geometric_kernels/backends/pytorch/gpytorch.py

```diff
@@ -19,14 +20,36 @@ def __init__(self, kernel: BaseGeometricKernel, **kwargs):
 
         self._kernel = kernel
 
+        params, state = self._kernel.init_params_and_state()
+
+        self.state = state
+
+        self.lengthscale = torch.tensor(params["lengthscale"])
```

Comment:
I think that if we want to make the lengthscale parameter learnable within gpytorch, we also need to register it similarly as what was made below for the smoothness. 

We could also use the fact that the super class already has a lengthscale parameter by default and do, at the beginning of the initialization:
```
self.has_lengthscale = True
super(GPytorchGeometricKernel, self).__init__(has_lengthscale=True, ard_num_dims=None, **kwargs)
```
and then do
`self.lengthscale = torch.tensor(params["lengthscale"])`

Reply:
This is already done in the superclass (the base gpytorch kernel class). It has a static attribute `_has_lengthscale` which is set to True in `GPytorchGeometricKernel`.